pages/api/teams.ts

```diff
@@ -30,8 +30,8 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
       }
     });
 
-    return res.status(201).setHeader('Location', process.env.BASE_URL + '/api/teams/1').send(null);
+    return res.status(201).json({ message: 'Team created' });
   }
 
-  res.status(404).send(null);
+  res.status(404).json({ message: 'Team not found' });
```

Comment:
Codacy has a fix for the issue: [Replace `'Team·not·found'` with `"Team·not·found"`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689300)
```suggestion
  res.status(404).json({ message: "Team not found" });
```